src/test_workflow/README.md

```diff
@@ -193,6 +194,29 @@ At a high-level the benchmarking test workflow uses [opensearch-cluster-cdk](htt
 
 The benchmarking tests will be run nightly and if you have a feature in any released/un-released OpenSearch version that you want to benchmark periodically please create an issue and the team will reach out to you. In case you want to run the benchmarking test locally you can use `opensearch-cluster-cdk` repo to spin up an OS cluster in your personal AWS account and then use `opensearch-benchmark` to run performance test against it. The detailed instructions are available on respective GitHub repositories.
 
+#### Onboard feature for nightly benchmark runs:
```

Comment:
Is it worth to add a warning regarding the `65535` char limit on groovy?

Reply:
I am going to remove a bunch of jobs which will free up ~50% of the runs. 
I can give it a pass at the moment but will add the warning as and when we start getting more feature requests.